lib/diagnostic.md

```diff
@@ -21,7 +28,9 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+```html
+$("li.listerItem").siblings("FORM");
+```
```

Comment:
you don't need the li.listerItem .listerItem would suffice here.  Also form is not a sibling.

$('.listerItem').parent().siblings().last() or something similar